pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -89,6 +98,40 @@ func (m *kubeGenericRuntimeManager) generateLinuxContainerConfig(container *v1.C
 
 	lc.Resources.HugepageLimits = GetHugepageLimitsFromResources(container.Resources)
 
+	// Set memory.min and memory.high if MemoryQoS enabled with cgroups v2
+	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.MemoryQoS) &&
+		libcontainercgroups.IsCgroup2UnifiedMode() {
+		unified := map[string]string{}
+
+		if memoryRequest != 0 {
+			unified[cm.MemoryMin] = strconv.FormatInt(memoryRequest, 10)
+		}
+
+		memoryHigh := int64(0)
+		if memoryLimit != 0 {
+			memoryHigh = int64(float64(memoryLimit) * memoryThrottlingFactor)
```

Comment:
is it desirable to use the the same `memoryThrottlingFactor` for both calculating `memory.high` when memoryLimit exists as well as when memoryLimit does not exist and using node allocatable memory? Perhaps we want to have two different throttling factors? 

/cc @mrunalp

Reply:
I think one throttling factor makes it easier for understanding. When container sets the `memory_limit`, throttling is from the container perspective. If `memory limit` is not set, then it is from the global node perspective.

Reply:
I think this is okay for now but can revisit in next iteration.